runtimevar/blobvar/blobvar_test.go

```diff
@@ -105,6 +106,12 @@ func TestOpenVariable(t *testing.T) {
 	}
 	defer os.RemoveAll(dir)
 
+	// Convert dir to a URL path, adding a leading "/" if needed on Windows.
+	dirpath := filepath.ToSlash(dir)
+	if os.PathSeparator != '/' && !strings.HasPrefix(dirpath, "/") {
```

Comment:
Maybe add a comment saying that on Unix the path will be resolved to start with a '/'.

Reply:
Done.